modules/scf/chart.sh

```diff
@@ -63,9 +63,11 @@ if [ -n "$SCF_HELM_VERSION" ]; then
     popd
 fi
 
+rm -rf chart_url || true
+
 if echo "$SCF_CHART" | grep -q "http"; then
-    info "Downloading $SCF_CHART"
     wget "$SCF_CHART" -O chart
+    echo "$SCF_CHART" > chart_url
```

Comment:
we should differentiate it from the stratos chart, maybe `scf_chart_url`?

Reply:
Addressed in https://github.com/SUSE/catapult/pull/66/commits/1a970299781bc5b4c05f59f1f3a5ef347fb44f39